audio/src/ContentReactor.Audio/ContentReactor.Audio.Services/AudioService.cs

```diff
@@ -46,39 +47,46 @@ public AudioService(IBlobRepository blobRepository, IAudioTranscriptionService a
             var audioId = Guid.NewGuid().ToString();
 
             // create a blob placeholder (which will not have any contents yet)
-            var blob = BlobRepository.CreatePlaceholderBlob(AudioBlobContainerName, userId, audioId);
+            var blob = BlobRepository.CreatePlaceholderBlob(AudioBlobContainerName, audioId);
 
             // get a SAS token to allow the client to write the blob
-            var writePolicy = new SharedAccessBlobPolicy
+            BlobSasBuilder sasBuilder = new BlobSasBuilder
             {
-                SharedAccessStartTime = DateTime.UtcNow.AddMinutes(-5), // to allow for clock skew
-                SharedAccessExpiryTime = DateTime.UtcNow.AddHours(24),
-                Permissions = SharedAccessBlobPermissions.Create | SharedAccessBlobPermissions.Write
+                StartsOn = DateTime.UtcNow.AddMinutes(-5),
```

Comment:
The clock skew comment seems helpful.   We may want to keep that here to explain why we're rolling back by 5 minutes.

Reply:
Resolved